central/notifier/service/service_impl_test.go

```diff
@@ -12,14 +12,15 @@ import (
 	"github.com/stackrox/rox/pkg/errox"
 	reporterMocks "github.com/stackrox/rox/pkg/integrationhealth/mocks"
 	"github.com/stackrox/rox/pkg/notifier/mocks"
+	"github.com/stackrox/rox/pkg/notifiers"
+	notifiersMocks "github.com/stackrox/rox/pkg/notifiers/mocks"
 	"github.com/stackrox/rox/pkg/sac"
 	"github.com/stackrox/rox/pkg/secrets"
 	"github.com/stretchr/testify/suite"
 	"go.uber.org/mock/gomock"
 )
 
 func TestNotifierService(t *testing.T) {
-	t.Parallel()
```

Comment:
why we need to remove it?

Reply:
I initially replaced the `generic` notifier creator in the `Registry`. With parallel execution that would make problems and potential race conditions.

I have changed tests to use fake notifier types (we don't have any validation for that) for each test. That should be fine for parallel execution.